### ISSUE TITLE
digest-headers: examples in 10.4 incomplete

### ISSUE THREAD

**Issue by reschke** [httpwg/http-extensions#1358]
```
The request contains a Digest field calculated on the enclosed representation.

It also includes an Accept-Encoding: br header field that advertises the client supports brotli encoding.

The response includes a Content-Encoding: br that indicates the selected representation is brotli encoded. The Digest field-value is therefore different compared to the request.

The response body is displayed as a base64-encoded string because it contains non-printable characters.
```
It's not self-evident that the response body actually contains a representation of the resource. It could be a status message as well. The prose should clarify that.
```
Request:

PUT /items/123
```

- [x] Missing Host: 

```
Content-Type: application/json
Accept-Encoding: br
Digest: sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=

{"hello": "world"}
```

- [x] Suggest to add content-length for clarity.

```
Response:

```

- [x] Missing status line.

```
Content-Type: application/json
Content-Encoding: br
Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=
```

- [x] Suggest to add Content-Location so it's clear that the response body describes the current representation.

```

iwiAeyJoZWxsbyI6ICJ3b3JsZCJ9Aw==
```

**Comment by ioggstream** [comment]
@reschke does this wording clarify better? https://github.com/httpwg/http-extensions/blob/master/draft-ietf-httpbis-digest-headers.md#client-and-server-provide-full-representation-data

The PR above addresses the adding of Host and Content-Location. 

Thanks for all the support!

**Comment by reschke** [comment]
10.6 is lacking Content-Location as well...

**Comment by ioggstream** [comment]
fixed in 121c7a5
